app/views/CalculationLPP2View.scala.html

```diff
@@ -87,13 +87,12 @@ <h1 class="govuk-heading-l govuk-!-margin-bottom-5">@messages("calculation.headi
 
   </dl>
 
-  @if(isTTPActive) {
-    @insetText(InsetText(
-      id = Some("ttp-inset-text"),
-      content = HtmlContent(getMessage("calculation.ttp.insetText"))))
-  }
-
   @if(isEstimate) {
+    @if(isTTPActive) {
+        @insetText(InsetText(
+            id = Some("ttp-inset-text"),
+            content = HtmlContent(getMessage("calculation.ttp.insetText"))))
+    }
     <h2 class="govuk-heading-s">@messages("calculation.h2.estimates")</h2>
     @if(isTTPActive) {
       <p class="govuk-body">@getMessage("calculation.p.3.estimates.lpp.ttpActive")</p>
```

Comment:
Just thinking if you want to add a class to this so you can check if _any_ TTP content exists?